### DISCUSSION TITLE
Add to osu lazer tablet settings antichatter (like Devocub antichatter)

### DISCUSSION THREAD

**Discussion by kostya1F634** [ppy/osu#18765] [Ideas] [answered]
Many players, including me, use third-party drivers because of the antichatter function. Therefore, I would like a similar function to be implemented inside the osu client. This is a very useful feature.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/discussions/15306

**Reply by kostya1F634** [reply]
Thanks.
